src/EluvioLive.js

```diff
@@ -43,40 +43,35 @@ class EluvioLive {
     this.client.ToggleLogging(false);
   }
 
+  async InitNew() {
+    this.client = await ElvClient.FromConfigurationUrl({
+      configUrl: this.configUrl,
+    });
+    let wallet = this.client.GenerateWallet();
+    const mnemonic = wallet.GenerateMnemonic();
+    const signer = wallet.AddAccountFromMnemonic({ mnemonic });
+    const privateKey = signer.privateKey;
+    const address = signer.address;
+
+    this.client.SetSigner({ signer });
+    this.client.ToggleLogging(false);
+
```

Comment:
For this we need a few things:
- make sure to create a "UserWallet" contract for this user (required in the current data model)
- take an optional tenant ID and set the current tenant ID for this user